src/webgpu/api/operation/rendering/depth.spec.ts

```diff
@@ -84,11 +92,7 @@ g.test('depth_compare_func')
             [[location(0)]] var<out> color : vec4<f32>;
 
             [[stage(vertex)]] fn main() -> void {
-              const pos : array<vec2<f32>, 3> = array<vec2<f32>, 3>(
-                  vec2<f32>(-1.0, -3.0),
-                  vec2<f32>(3.0, 1.0),
-                  vec2<f32>(-1.0, 1.0));
-              Position = vec4<f32>(pos[VertexIndex], 0.5, 1.0);
+              Position = vec4<f32>(0.5, 0.5, 0.5, 1.0);
```

Comment:
I just realized I think(?) the middle of the screen would have been 0,0 not 0.5,0.5. Sorry. There should be nothing wrong with 0.5,0.5 though so just leaving it.

Reply:
Ah... I didn't realize that either...

Reply:
Well since we are on this topic. What is the equivalent for gl_PointSize in webgpu/wgsl ? (didn't find one)

Reply:
There is no gl_PointSize in WebGPU, you have to use instanced rectangles if you want bigger points. This is to avoid the portability issues with larger points like in WebGL. (On some AMD they clip if the center of the point goes off the screen, so half of the large point disappears suddenly)